### PR TITLE
fix(Error message) Added error message to CgError, fixes #1593

### LINKED ISSUE #1593: No error message on missing working dir
## Description

When running CG clean, if there is a problem with a working directory, there is no error massage when the exception is raised. This comes from. This makes it a little bit harder to quickly troubleshoot as it looks like a bug.

## Suggested solution

Add an error message when raising the exception

## This can be closed when
- [ ] #1594

### PR DESCRIPTION
**Pull request by pbiology**
## Description

This just adds an error message when a working dir cannot be found for easier troubleshooting.

### Fixed

- Error message when raising CgError

### How to prepare for test
- [ ] ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] paxa the environment: `paxa`
- [ ] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh workdir_error_msg`

### How to test
- [ ] do `cg clean balsamic-past-run-dirs `date -d -10days +%Y-%m-%d` -y -d`

### Expected test outcome
- [ ] Error message should be printed
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/meta/workflow/analysis.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def check_analysis_ongoing(self, case_id: str) -> None:
     def verify_case_path_exists(self, case_id: str) -> None:
         if not self.get_case_path(case_id=case_id).exists():
             LOG.error("Working directory path for %s does not exist", case_id)
-            raise CgError()
+            raise CgError("No working directory for %s exists", case_id)
```
Comment (henrikstranneheim) [inline cg/meta/workflow/analysis.py:78]: On second thought
```suggestion
            raise CgError(f"No working directory for {case_id} exists")
```

Comment (karlnyr) [reply cg/meta/workflow/analysis.py:78]: Should probably apply this to the line above as well then 👍

Comment (pbiology) [reply cg/meta/workflow/analysis.py]: Updated both in 3a22bdf5

**Comment by pbiology** [comment]
Deployed
Screenshot 2022-10-10 at 14 24 26

**Pull request merged**